frontend/src/Content.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react'
+import { useGeneralContext } from './contexts/generalState'
+import getBackgroundImages from './api/background'
+import Footer from './components/Footer/Footer'
+import Router from './Router'
+import Sidebar from './components/Sidebar/Sidebar'
+
+function Content() {
+  const { setBackgroundImages } = useGeneralContext()
+  const [loading, setLoading] = useState<boolean>(true)
+
+  useEffect(() => {
+    ;(async () => {
+      const bgImages = await getBackgroundImages()
+      setBackgroundImages(bgImages)
+      setLoading(false)
+    })()
+  }, [])
+
+  return loading ? (
+    <div className="spinner-border" role="status" />
```

Comment:
I know this div was just moved, but maybe you could choose a better naming for className. You should not mention in a class name any styling properties (like border, color, etc). This one could simply be `spinner`

Reply:
the className is for bootstrap :)